timeseries/tests/test_Interface.py

```diff
@@ -0,0 +1,135 @@
+from Interface import SizedContainerTimeSeriesInterface
+from pytest import raises
+import reprlib
+import numpy as np
+import collections
+
+#define a concrete SizedContainerTimeSeriesInterface
+class concrete_SizedContainerTimeSeriesInterface(SizedContainerTimeSeriesInterface):
+
+    def __init__(self, values, times=None):
+        if not isinstance(values, collections.Sequence):
+            raise TypeError("Input values must be Sequence")
+        if times is not None:
+            if not isinstance(times, collections.Sequence):
+                raise TypeError("Input times must be Sequence")
+            if len(times) != len(values):
+                raise ValueError("Input values sequence and times sequence must have the same length")
+            self._time = list(times)
+        else:
+            self._time = list(range(len(values)))
+        self._value = list(values)
+        self._timeseries = list(zip(self._time, self._value))
+
+    def __setitem__(self, index, value):
+        pass
+
+    def __add__(self, other):
+        pass
+
+    def __sub__(self, other):
+        pass
+
+    def __mul__(self, other):
+        pass
+
+    def __eq__(self, other):
+        pass
+
+    def __abs__(self):
+        pass
+
+    def __bool__(self):
+        pass
+
+    def __neg__(self):
+        pass
+
+    def __pos__(self):
+        pass
+
+    def interpolate(self, time_seq):
+        pass
+
+    def mean(self):
+        return np.mean(self._value)
+
+    def std(self):
+        return np.std(self._value)
+
+    def __len__(self):
+        return len(self._value)
+
+    def __getitem__(self, index):
+        return self._timeseries[index]
+
+    def __contains__(self, val):
+        return val in self._value
+
+    def values(self):
+        return self._value
+
+    def times(self):
+        return self._time
+
+    def items(self):
+        return self._timeseries
+
+    def __str__(self):
+        class_name = type(self).__name__
+        return class_name + '(' + reprlib.repr([item for item in self._timeseries])
+
+    def __repr__(self):
+        class_name = type(self).__name__
+        return class_name + '(' + reprlib.repr([item for item in self._timeseries]) + '), length={}'.format(len(self))
+
+#test mean
+def test_mean():
```

Comment:
maybe we can put several test functions into one, something like test_basic(), test_mean_std() etc., and add comments therein to tell what you are testing for